Monika After Story/game/script-topics.rpy

```diff
@@ -8712,9 +8712,14 @@ label monika_player_appearance:
 
                 call monika_player_appearance_monika_height
 
-                m 3esc "The tallest girl in the literature club was Yuri-- and just barely, at that. She was only a few inches taller than me, I don’t consider that much of a height advantage at all!"
-                m 3esd "Anyway, dating a tall [guy] like you only has one disadvantage, [player]..."
-                m 1hub "You’ll have to lean down to kiss me!"
+                if persistent._mas_pm_units_height_metric:
```

Comment:
Just a little thing, you can minimize the amount to repeat here if only the `m 3esc "The tallest girl in the literature club was Yuri-- (and the rest)` line was inside the `if`/`else` blocks, since the other two lines are unchanged, we don't need to put those into the blocks.

Reply:
Okay, thanks